### ISSUE TITLE
Cannot read property edges of undefined

### ISSUE THREAD

**Issue by zephraph** [intuit/auto#341]
**Describe the bug**

Trying to deploy `relay-compiler-language-typescript` [fails](https://travis-ci.org/relay-tools/relay-compiler-language-typescript/builds/521006779#L548) with the error `Cannot read property 'edges' of undefined`. 

```
TypeError: Cannot read property 'edges' of undefined
    at /home/travis/.npm/_npx/7088/lib/node_modules/auto/dist/release.js:269:78
    at Array.map (<anonymous>)
    at /home/travis/.npm/_npx/7088/lib/node_modules/auto/dist/release.js:261:53
    at Array.map (<anonymous>)
    at Release.<anonymous> (/home/travis/.npm/_npx/7088/lib/node_modules/auto/dist/release.js:260:30)
    at step (/home/travis/.npm/_npx/7088/lib/node_modules/auto/dist/release.js:49:23)
    at Object.next (/home/travis/.npm/_npx/7088/lib/node_modules/auto/dist/release.js:30:53)
    at fulfilled (/home/travis/.npm/_npx/7088/lib/node_modules/auto/dist/release.js:21:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by hipstersmoothie** [comment]
I tested the change on the mentioned repo and got the following output:

```
➜ ../github-release-ts/dist/bin/auto.js release --dry-run
ℹ  info      Last used release: dd5c473f8145dba7d8e357c97f9cfe24fc19d76d
bailed
bailed
ℹ  info      Using release notes:
#### 🐛  Bug Fix

- [WIP] Update example app with latest changes. [#79](https://github.com/relay-tools/relay-compiler-language-typescript/pull/79) ([@alloy](https://github.com/alloy))
- fix typing  ([@martin.artola@swisslife.ch](https://github.com/martin.artola@swisslife.ch))
- Upgrade to Relay v2.0.0 #85  ([@martin@artola.info](https://github.com/martin@artola.info))
- support --noImplicitAny [#83](https://github.com/relay-tools/relay-compiler-language-typescript/pull/83) ([@koddsson](https://github.com/koddsson) eloy.de.enige@gmail.com [@timjacobi](https://github.com/timjacobi))
- Add support for noFutureProofEnums [#82](https://github.com/relay-tools/relay-compiler-language-typescript/pull/82) (eloy.de.enige@gmail.com)
- Dont require relay runtime string [#77](https://github.com/relay-tools/relay-compiler-language-typescript/pull/77) ([@koddsson](https://github.com/koddsson))
- Use `optionalInputFields` instead of `inputFieldWhiteList` [#78](https://github.com/relay-tools/relay-compiler-language-typescript/pull/78) ([@koddsson](https://github.com/koddsson))
- Warning no longer valid; relay-runtime@1.7.0 available [#76](https://github.com/relay-tools/relay-compiler-language-typescript/pull/76) ([@chrisdostert](https://github.com/chrisdostert))
- Removes package-lock.json. [#74](https://github.com/relay-tools/relay-compiler-language-typescript/pull/74) ([@ashfurrow](https://github.com/ashfurrow))
- Sets up TSLint, precommit hooks [#72](https://github.com/relay-tools/relay-compiler-language-typescript/pull/72) ([@ashfurrow](https://github.com/ashfurrow))
- Add README section about react-hot-loader problem [#68](https://github.com/relay-tools/relay-compiler-language-typescript/pull/68) ([@kalekseev](https://github.com/kalekseev) [@alloy](https://github.com/alloy))
- fix oboe for source location line [#65](https://github.com/relay-tools/relay-compiler-language-typescript/pull/65) ([@mattkrick](https://github.com/mattkrick))
- Update README instructions about relay version [#69](https://github.com/relay-tools/relay-compiler-language-typescript/pull/69) ([@kalekseev](https://github.com/kalekseev))
- [example] Upgrade @types/react-relay and @react/relay-runtime [#62](https://github.com/relay-tools/relay-compiler-language-typescript/pull/62) ([@alloy](https://github.com/alloy))
- [package] Prepare for v1.1.0  ([@alloy](https://github.com/alloy))
- [plugin] Type fragment refs as any when not using a single artifact dir. [#61](https://github.com/relay-tools/relay-compiler-language-typescript/pull/61) ([@alloy](https://github.com/alloy))
- Emit combined operation typings that can be used with generics [#60](https://github.com/relay-tools/relay-compiler-language-typescript/pull/60) ([@alloy](https://github.com/alloy))
- [plugin] Make plugin work with Relay < 1.6 [#57](https://github.com/relay-tools/relay-compiler-language-typescript/pull/57) ([@alloy](https://github.com/alloy))
- [CI] Adds badge to readme. [#55](https://github.com/relay-tools/relay-compiler-language-typescript/pull/55) ([@ashfurrow](https://github.com/ashfurrow))
- Adds continuous integration tests. [#54](https://github.com/relay-tools/relay-compiler-language-typescript/pull/54) ([@ashfurrow](https://github.com/ashfurrow))
- Adds type-checking support for fragment references. [#53](https://github.com/relay-tools/relay-compiler-language-typescript/pull/53) ([@alloy](https://github.com/alloy))
- [plugin] Port test for recursive fragments from upstream. [#52](https://github.com/relay-tools/relay-compiler-language-typescript/pull/52) ([@alloy](https://github.com/alloy))
- [plugin] Port stack overflow fix. [#51](https://github.com/relay-tools/relay-compiler-language-typescript/pull/51) ([@alloy](https://github.com/alloy))
- Release 0.9.1 [#50](https://github.com/relay-tools/relay-compiler-language-typescript/pull/50) ([@alloy](https://github.com/alloy))
- Fix property optionalism within Input Objects [#47](https://github.com/relay-tools/relay-compiler-language-typescript/pull/47) ([@ckknight](https://github.com/ckknight))
- Update to latest relay [#49](https://github.com/relay-tools/relay-compiler-language-typescript/pull/49) ([@alloy](https://github.com/alloy))
- [README] Document need to output ES modules  ([@alloy](https://github.com/alloy))
- [package] Update to latest Relay version of language-plugin [#39](https://github.com/relay-tools/relay-compiler-language-typescript/pull/39) ([@alloy](https://github.com/alloy))
- Update to latest relay [#38](https://github.com/relay-tools/relay-compiler-language-typescript/pull/38) ([@alloy](https://github.com/alloy))
- Adds an ambient declaration for the expect extension [#34](https://github.com/relay-tools/relay-compiler-language-typescript/pull/34) ([@orta](https://github.com/orta))
- Release prep [#31](https://github.com/relay-tools/relay-compiler-language-typescript/pull/31) ([@alloy](https://github.com/alloy))
- Bring artifacts up to date with DT part 2 [#30](https://github.com/relay-tools/relay-compiler-language-typescript/pull/30) ([@alloy](https://github.com/alloy))
- Bring artifacts up to date with dt [#28](https://github.com/relay-tools/relay-compiler-language-typescript/pull/28) ([@alloy](https://github.com/alloy))
- [TypeGenerator] Use relative fragment reference imports. [#24](https://github.com/relay-tools/relay-compiler-language-typescript/pull/24) ([@alloy](https://github.com/alloy))
- [example] Import generated artifacts.  ([@alloy](https://github.com/alloy))
- [TS generator] Make sure the ref names are declared [#16](https://github.com/relay-tools/relay-compiler-language-typescript/pull/16) ([@kastermester](https://github.com/kastermester))
- Added MIT license file [#14](https://github.com/relay-tools/relay-compiler-language-typescript/pull/14) ([@kastermester](https://github.com/kastermester) [@alloy](https://github.com/alloy))
- Fixes issue #11 [#13](https://github.com/relay-tools/relay-compiler-language-typescript/pull/13) ([@kastermester](https://github.com/kastermester))
- [example] Use all available typings thus far and able to launch server.  ([@alloy](https://github.com/alloy))
- [package] Bump to 1.0.0-alpha.2  ([@alloy](https://github.com/alloy))
- [formatGeneratedModule] Remove Flow import syntax.  ([@alloy](https://github.com/alloy))
- [example] Make sure only 1 graphql package is installed.  ([@alloy](https://github.com/alloy))
- [README] More install instructions.  ([@alloy](https://github.com/alloy))
- [example] Configure dependencies.  ([@alloy](https://github.com/alloy))
- [package] Rename to relay-compiler-language-typescript  ([@alloy](https://github.com/alloy))
- [example] Import Relay example  ([@alloy](https://github.com/alloy))
- [prettier] Run  ([@alloy](https://github.com/alloy))
- [FindGraphQLTags] Add first test.  ([@alloy](https://github.com/alloy))
- [PluginInterface] Add file path to tag finder.  ([@alloy](https://github.com/alloy))
- [PluginInterface] Add type generator options.  ([@alloy](https://github.com/alloy))
- [TypeGenerator] Disable fragment imports for now.  ([@alloy](https://github.com/alloy))
- [test] Setup test suite using fixtures from relay-compiler  ([@alloy](https://github.com/alloy))
- [package] Build task  ([@alloy](https://github.com/alloy))
- [package] Remove unused dependencies  ([@alloy](https://github.com/alloy))
- [package] Add prettier with default settings.  ([@alloy](https://github.com/alloy))
- [plugin] Initial implementation  ([@alloy](https://github.com/alloy))
- [types] Remove all unused typings.  ([@alloy](https://github.com/alloy))
- Import plugin interface [#9](https://github.com/relay-tools/relay-compiler-language-typescript/pull/9) ([@alloy](https://github.com/alloy))
- [test] Import fixtures that mirror Flow tests upstream.  ([@alloy](https://github.com/alloy))

#### ⚠️  Pushed to master

- v2.0.0  ([@alloy](https://github.com/alloy))
- Merge branch 'artola-master'  ([@sibelius](https://github.com/sibelius))
- feat(merge): merge with master  ([@sibelius](https://github.com/sibelius))
- v1.1.0  ([@alloy](https://github.com/alloy))
- [README] Small tweak  ([@alloy](https://github.com/alloy))
- [README] Add link to babel ES module transform.  ([@alloy](https://github.com/alloy))
- [README] Clarify what packages to use.  ([@alloy](https://github.com/alloy))
- 0.9.0  ([@alloy](https://github.com/alloy))
- [package] Switch to prebuilt Relay packages.  ([@alloy](https://github.com/alloy))
- [README] Update links to artifacts.  ([@alloy](https://github.com/alloy))
- Added previously ignored src/bin files and missing test files  ([@kastermester](https://github.com/kastermester))
- Added previously ignored src/bin files and missing test files  ([@kastermester](https://github.com/kastermester))
- More tests and support for references added  ([@kastermester](https://github.com/kastermester))
- Unignore src/bin  ([@khs@secoya.dk](https://github.com/khs@secoya.dk))
- Initial POC done  ([@kastermester](https://github.com/kastermester))
- Began typing up the graphql-compiler package.  ([@khs@secoya.dk](https://github.com/khs@secoya.dk))

#### Authors: 15

- Eloy Durán ([@alloy](https://github.com/alloy))
- [@martin.artola@swisslife.ch](https://github.com/martin.artola@swisslife.ch)
- [@martin@artola.info](https://github.com/martin@artola.info)
- Kristján Oddsson ([@koddsson](https://github.com/koddsson))
- eloy.de.enige@gmail.com
- Tim Jacobi ([@timjacobi](https://github.com/timjacobi))
- Chris Dostert ([@chrisdostert](https://github.com/chrisdostert))
- Ash Furrow ([@ashfurrow](https://github.com/ashfurrow))
- Konstantin Alekseev ([@kalekseev](https://github.com/kalekseev))
- Matt Krick ([@mattkrick](https://github.com/mattkrick))
- Cameron Knight ([@ckknight](https://github.com/ckknight))
- Orta ([@orta](https://github.com/orta))
- Kaare Hoff Skovgaard ([@kastermester](https://github.com/kastermester))
- [@sibelius](https://github.com/sibelius)
- [@khs@secoya.dk](https://github.com/khs@secoya.dk)
ℹ  info      Publishing v2.2.0 to GitHub.
```

**Comment by zephraph** [comment]
Oh, thanks man!

**Comment by hipstersmoothie** [comment]
If you release the latest tag as "Latest Release" the changelog will be a bit smaller!

No problem!

**Comment by adierkens** [comment]
:rocket: Issue was released in v4.9.3 :rocket:
